docs/src/references/custom-fragments.md

```diff
@@ -0,0 +1,102 @@
+# Custom Fragments
+
+**Custom-Fragments** are snippets of meta-information, provided by processes running on the **main-device** or any **child-device**;
+that meta-information can be e.g. the serial-number of the device, kinds of sensors the device provides, or signal strength of the device's LTE/3G connection.
+
+* all **custom-fragments** are provided to **thin-edge** via MQTT topics
+* the cloud mapper sends all **custom-fragments** to the cloud's corresponding **device-twins**
+* processes running on the **main-device** and **child-devices** can consume and use the provided **custom-fragments** from MQTT (when not restricted for any security reason)
+
+## Custom-Fragments on MQTT
+
+All **custom-fragments** are provided and consumed via MQTT.
+
+
+#### Topics
+
+```cjson
+/tedge/inventory/<device-id>/<identifider>
+```
+
+|Reference     | Description|
+| ------------ | ---------- |
+|`device-id`   | if the custom-fragment is meant for a **child-device**, it is the corresponding **child-id**; or `main` if it is meant for the **main-device**|
```

Comment:
Using an explicit `main` id is a really good idea, removing many issues we currently have with topics that need to be parsed differently for the main and for child devices.